app/assets/javascripts/initializers/initializeUserProfilePage.js

```diff
@@ -1,11 +1,23 @@
 'use strict';
 
 function initializeUserProfilePage() {
-  if (document.getElementById('user-profile-dropdown')) {
-    document.getElementById('user-profile-dropdown').onclick = () => {
-      document
-        .getElementById('user-profile-dropdownmenu')
-        .classList.toggle('showing');
-    };
+  if(document.querySelector('.profile-dropdown')){
```

Comment:
this can just be:

```javascript
const profileDropdownDiv = document.getElementsByClassName("profile-dropdown");
if (profileDropdownDiv) {
}
```

this way you avoid querying twice and you use `getElementsByClassName` which is faster than `querySelector`.

Also keep an eye on spacing :) we're not super strict on rules but since you're already working on this file could you also "clean it" and modernize it?